staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -261,11 +261,19 @@ func patchResource(
 			// 3. ensure no conflicts between the two patches
 			// 4. apply the #1 patch to the currentJS object
 
-			// Since the patch is applied on versioned objects, we need to convert the
-			// current object to versioned representation first.
-			currentVersionedObject, err := unsafeConvertor.ConvertToVersion(currentObject, kind.GroupVersion())
-			if err != nil {
-				return nil, err
+			var currentVersionedObject runtime.Object
+			var err error
```

Comment:
no need for the err here, `currentVersionedObject, err := ...` in the if should do.

Reply:
This doesn't work in my environment.
https://play.golang.org/p/w6VFItDR4h

Reply:
right, currentVersionedObject is shared. But then at least move the `var err error` just above the `currentVersionedObject, err` line inside the if. Anyway, cosmetical nit.